sql/core/src/test/scala/org/apache/spark/sql/CharVarcharTestSuite.scala

```diff
@@ -603,6 +611,26 @@ class FileSourceCharVarcharTestSuite extends CharVarcharTestSuite with SharedSpa
       }
     }
   }
+
+  // TODO: Move this test to super after SPARK-33875 implements DESCRIBE COLUMN for v2
```

Comment:
Please make this as IDed TODO like `TODO(SPARK-33875)` and add some description about `Move this test...` to SPARK-33875's description.